templates/partials/newsletter-form.html

```diff
@@ -1,51 +1,61 @@
 <form
-	class="n-newsletter-signup__form"
-	data-component="n-newsletter-signup__form"
-	data-newsletter-id="{{id}}"
-	action="
-		{{~#if userIsSubscribed~}}
-			{{unsubscribeAction}}
-		{{~else~}}
-			{{subscribeAction}}
-		{{~/if~}}
-		"
-	method="POST"
-	>
-	{{> n-myft-ui/components/csrf-token/input}}
-	<button
-		class="n-newsletter-signup__submit"
-		aria-label="
-			{{~#if userIsSubscribed~}}
-				Unsubscribe from {{name}}
-			{{~else~}}
-				One-Click Sign Up subscribe to {{name}}
-			{{~/if~}}
-		"
-		title="
-			{{~#if userIsSubscribed~}}
-				Unsubscribe from {{name}}
-			{{~else~}}
-				Subscribe to {{name}}
-			{{~/if~}}
-		"
-		data-trackable="
-			{{~#if userIsSubscribed~}}
-				newsletter-unsubscribe
-			{{~else~}}
-				newsletter-subscribe
-			{{~/if~}}
-		"
-		type="submit"
-		aria-describedby="feedback-message__newsletter-{{id}}"
-		{{#if updatingPreference}}disabled="disabled" aria-disabled="true"{{/if}}
-		>
-
-			{{#if updatingPreference}}
-				Updating<span class="o-normalise-visually-hidden">&nbsp;{{name}} subscription</span>…
-			{{else if userIsSubscribed}}
-				Unsubscribe<span class="o-normalise-visually-hidden">&nbsp;from {{name}}</span>
-			{{else}}
-				One-Click Sign Up<span class="o-normalise-visually-hidden">&nbsp;to {{name}}</span>
-			{{/if}}
-		</button>
+  class="n-newsletter-signup__form"
+  data-component="n-newsletter-signup__form"
+  data-newsletter-id="{{id}}"
+  action="
+    {{~#if userIsSubscribed~}}
+      {{unsubscribeAction}}
+    {{~else~}}
+      {{subscribeAction}}
+    {{~/if~}}
+  "
+  method="POST"
+>
+  {{> n-myft-ui/components/csrf-token/input}}
+<div
+  class="ft-concept-button ft-concept-button--follow ft-concept-button--standard"
+  data-ft-concept-button-aria-live-pressed-text="Now following {{name}} newsletter"
+  data-ft-concept-button-aria-live-unpressed-text="No longer following {{name}} newsletter"
+  data-ft-concept-button-aria-label-pressed-text="Stop following {{name}} newsletter"
+  data-ft-concept-button-aria-label-unpressed-text="Follow {{name}} newsletter"
```

Comment:
**Change request:** The wording on this about "following" a newsletter is still incorrect as we don't "follow" newsletters. This language is inconsistent with the language used elsewhere in this feature regarding "Sign up", "Subscribe" etc